platforms/android/library-compose/build.gradle

```diff
@@ -9,6 +11,16 @@ if (project.hasProperty("coverage")) {
     apply plugin: 'jacoco'
 }
 
+apply plugin: "com.vanniktech.maven.publish.base"
```

Comment:
Apply plugin statements should be avoided, can you use this plugin from the `plugins{}`` block instead?

```
id 'com.vanniktech.maven.publish'
```

Reply:
Good point, I've updated this in the view library as well.